server/src/infra/repositories/filesystem.provider.ts

```diff
@@ -64,7 +64,13 @@ export class FilesystemProvider implements IStorageRepository {
   }
 
   async unlink(file: string) {
-    await fs.unlink(file);
+    await fs.unlink(file).catch((err) => {
```

Comment:
Why not just use a normal try/catch here?

Reply:
No particular reason. I can make it a normal try/catch